invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -291,8 +291,15 @@ def _format_thesis(thesis):
         if not isinstance(thesis, dict):
             return thesis
 
-        fields = ["university", "department", "type"]
-        formatted = [thesis.get(field, "") for field in fields]
+        department = thesis.get("department")
+        department = f"({department})" if department else ""
+        date_submitted = thesis.get('date_submitted')
+        date_defended = thesis.get('date_defended')
+        submitted = f"{_('Submitted: ')}{date_submitted}" if date_submitted else ""
+        defended = f"{_('Defended: ')}{date_defended}" if date_defended else ""
+        return (f"{thesis.get('university')} {department}, {thesis.get('type')}, "
```

Comment:
Maybe add a comment that university is deprecated and will be removed in a future release

Reply:
Sorry..please ignore